css/style.css

```diff
@@ -16,4 +19,144 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*Header*/
+
+header {
+  display: flex;
+  padding: 0 5rem;
+  justify-content: space-between;
+}
+
+/*Karma Logo*/
+
+header img {
+  height: 3rem;
+  width: 3rem;
+  margin-top: 10px;
+}
+
+/*Navigation*/
+
+ul {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+}
+
+li {
+  padding: 0.5rem;
+}
+
+a {
+  color: gray;
+  text-decoration: none;
+}
+
+a:hover {
+  color: var(--orange-karma);
+}
+
+/*Intro Image*/
+
+.intro {
+  display: flex;
+  background-image: url("../img/first-background.jpg");
+  padding: 15.5rem 0;
+  color: white;
+  flex-direction: column;
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  margin-bottom: 6rem;
+}
+
+/*Intro Text*/
+
+h1,
+h2,
+h3,
+.intro p {
+  font-weight: lighter;
+  text-align: center;
+}
+
+h1 {
+  font-size: 2.5rem;
+  margin: 0;
+}
+
+.intro p {
+  font-size: 1.5rem;
+}
 
+/*Intro Button*/
+
+button {
+  padding: 0.75rem 1.25rem;
+  background-color: #f26639;
+  border: none;
+  border-radius: 5px;
+  font-size: 0.75rem;
+  margin-top: 2rem;
+}
+
+/*Main*/
+
+/*Main Text*/
+
+h2 {
+  font-size: 2rem;
+}
+
+p {
+  font-size: 1.4rem;
+}
+
+/*Karma Icon Style*/
+
+.karma_icon_style {
+  display: flex;
+  justify-content: space-evenly;
+  margin-right: 3rem;
+}
+
+.karma_icon_direction {
+  display: flex;
+  flex-direction: column;
+}
+
+.karma_icon_images {
+  height: 50%;
+}
+
+hr {
+  width: 90%;
+  opacity: 25%;
+}
+
+/*footer*/
+
+/*Social Media Style*/
+
+.social_media {
+  display: flex;
+  justify-content: center;
+}
+
+.social_media_images {
+  display: flex;
+  height: 1.5rem;
+  padding: 0.5rem;
+  margin: 0rem 0.5rem;
+  border: 2px solid var(--cream-karma);
+  border-radius: 30px;
+}
+
```

Comment:
As you might have noticed, the images do not show upon the page correctly -- this is because while the div they are in is sized, the images themselves are not. What you want, is to constrain them to follow the size of the div, for example:

```css
.social_media_images img {
  height: 100%;
}
```

Just make sure you only fix one dimension, not both! This will allow the image to retain the original ratios and not get distorted.

Reply:
Hi Joanna, I added the lines to my CSS file but I couldn't see any difference. Are you referring to just the social media images or all the images?

Reply:
Never mind, I understand now. I got confused because I removed display flex from social_media_images. It had the same effect as adding the lines you suggested.

Reply:
Oh, I think I made this mistake the other way around! I added the additional img styling first, and then simply checked removing `display: flex` didn't break anything...